src/AspNetCore/Src/inprocessapplication.cxx

```diff
@@ -107,6 +135,82 @@ http_read_request_bytes(
     _In_ IHttpContext* pHttpContext,
     _In_ CHAR* pvBuffer,
     _In_ DWORD cbBuffer,
+    _In_ DWORD* pDwBytesReceived,
+    _In_ BOOL* pfCompletionPending
+)
+{
```

Comment:
as it is exported method, it's better to check the input parameters, e.g., NULL

Reply:
I'll add this check for now. Should we add it everywhere?